### PR TITLE
Use API instead of mocked data to get proctoring settings.

### PR DESCRIPTION
**Pull request by UvgenGen**
[EDUCATOR-5803](https://openedx.atlassian.net/browse/EDUCATOR-5803) Use API instead of mocked data to get proctoring settings.

### CODE REVIEW

**Review by viktorrusakov — APPROVED** [review]

File: src/data/api.js [hunk 1]
```diff
@@ -54,24 +54,8 @@ export async function endExamWithFailure(attemptId, error) {
...
-        exam_register_endpoint: '{add endpoint to SoftwareSecure}',
-        exam_sponsor: '{add SoftwareSecure sponsor}',
-        organization: '{add SoftwareSecure organization}',
-        software_download_url: '{add SoftwareSecure download url}',
-      },
-      default: 'software_secure',
-    },
-  };
+export async function fetchProctoringSettings(examId) {
+  const url = new URL(`${getConfig().LMS_BASE_URL}/api/edx_proctoring/v1/settings/exam_id/${examId}/`);
```
Comment (idegtiarov) [inline src/data/api.js]: We've updated that URL on the backend, so please do the same change here:
```suggestion
  const url = new URL(`${getConfig().LMS_BASE_URL}/api/edx_proctoring/v1/proctored_exam/settings/exam_id/${examId}/`);
```

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: src/data/api.js (hunk 1 above)
Comment (UvgenGen) [reply src/data/api.js]: done

**Comment by idegtiarov** [comment]
@UvgenGen probably we will have to re-open PR since @ihor-romaniuk did the same to his. What do you think, will it help us to see that tests passed.

**Comment by UvgenGen** [comment]
@idegtiarov I will open PR again. After that, the tests must be passed.

**Review by idegtiarov — APPROVED** [review]
LGTM so far!  Please open PR to edX

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**